web/pdf_sidebar.js

```diff
@@ -90,9 +90,6 @@ var PDFSidebar = (function PDFSidebarClosure() {
     reset: function PDFSidebar_reset() {
       this.isInitialViewSet = false;
 
-      this.close();
```

Comment:
Sorry, this is probably not *entirely* relevant to this PR, and it concerns an edge-case anyway.

When looking through the code in `PDFSidebar`, it struck me that forcibly closing the sidebar during `PDFViewerApplication.close()` may not be what the user expects. This really only matters in default viewer, when the "Open" functionality is present.

If you feel that we shouldn't change this at all, or that it doesn't belong in this PR (which I'd totally understand!), I'll re-add these two lines!